src/server/Document/component.jsx

```diff
@@ -14,15 +15,21 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
   const headScript = helmet.script.toComponent();
   const serialisedData = JSON.stringify(data);
   const scriptsAllowed = !data.isAmp;
-  const scripts = assets.map(asset => (
-    <script
-      crossOrigin="anonymous"
-      key={asset}
-      type="text/javascript"
-      src={asset}
-      defer
-    />
-  ));
+  const scripts = (
+    <Fragment>
+      <IfAboveIE9>
+        {assets.map(asset => (
```

Comment:
On IE 9 using browserstack, I still see the scripts being added:

<img width="689" alt="Screen Shot page on browserstack using IE9" src="https://user-images.githubusercontent.com/3028997/55071062-88574480-507f-11e9-9e7b-d2839af216f2.png">

Reply:
Resolving this - when installing from branch again, these scripts are not being pulled in.

Reply:
Before:
<img width="1006" alt="Screen Shot with scripts" src="https://user-images.githubusercontent.com/3028997/55072586-6bbd0b80-5083-11e9-9504-c3076acde7a3.png">

After: 
<img width="1003" alt="Screen Shot without scripts" src="https://user-images.githubusercontent.com/3028997/55072590-6eb7fc00-5083-11e9-94a9-1546374b27f6.png">
👍